src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,48 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int buysum = 0;
+        int suplysum = 0;
+        String[] file = fileReader(fromFileName);
+        for (String lines : file) {
+            String[] splitfile = lines.split(",");
+            if (splitfile[0].equals("buy")) {
+                buysum += Integer.parseInt(splitfile[1]);
+            }
+            if (splitfile[0].equals("supply")) {
+                suplysum += Integer.parseInt(splitfile[1]);
+            }
+        }
+        int result = suplysum - buysum;
+        fileWriter(buysum,suplysum,result,toFileName);
+
+    }
+
+    private String[] fileReader(String fileName) {
```

Comment:
```suggestion
    private String[] readFromFile(String fileName) {
```
method names should be verbs